src/NCPkgFilterService.cc

```diff
@@ -0,0 +1,310 @@
+/****************************************************************************
+|
+| Copyright (c) [2002-2011] Novell, Inc.
```

Comment:
Not sure how useful the copyright that far in the past for a brand new file is...

Reply:
The file is largely copied from src/NCPkgFilterRepo.cc